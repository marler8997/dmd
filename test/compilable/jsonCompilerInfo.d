// PERMUTE_ARGS:
// REQUIRED_ARGS: -o- -Xq=compilerInfo -Xf${RESULTS_DIR}/compilable/jsonCompilerInfo.out
// POST_SCRIPT: compilable/extra-files/json-postscript.sh jsonCompilerInfo
