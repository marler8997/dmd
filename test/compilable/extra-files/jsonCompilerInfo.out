{
    "compilerInfo": {
        "binary": "VALUE_REMOVED_FOR_TEST",
        "supportsIncludeImports": true,
        "version": "VALUE_REMOVED_FOR_TEST"
    }
}